Gemfile

```diff
@@ -75,6 +79,7 @@ end
 #  Admin dashboard gems
 gem 'remotipart', github: 'mshibuya/remotipart'
 gem 'rails_admin', '>= 1.0.0.rc'
+gem 'remotipart', github: 'mshibuya/remotipart'
```

Comment:
It did not work to add the `ref: 'd9109eb032b938672f231155745549a180ee9032'`. I used the current version `1.2`.